process-create-data-repo-request

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+import re
+from datetime import datetime
+from optparse import OptionParser
+from os.path import expanduser
+from socket import setdefaulttimeout
+from typing import Optional, Any
+
+import github
+from github import Github
+
+from categories_map import CMSSW_CATEGORIES
+from cms_static import (
+    GH_CMSSW_REPO,
+    CREATE_REPO,
+)
+from github_utils import api_rate_limits
+
+setdefaulttimeout(120)
+
+
+# Processes a github issue to check if it is requesting the creation of a new data repo
+# If the issue is not requesting new repo, it ignores it.
+
+# -------------------------------------------------------------------------------
+# Global Variables
+# --------------------------------------------------------------------------------
+
+opts: Optional[Any] = None
+labels = []
+
+INVALID_REQUEST_MSG = "No category found for requested package {package}"
+EXISTS_MSG = "Requested repository {repo} already exists"
+ACK_MSG = (
+    "Request received. I will create the requested repository after one of the following approve "
```

Comment:
looks like we are not adding this comment. bot should add this for new issuess and it should mentioned that it will create the data repo `<repo-name>` once issue is fully signed

Reply:
done